tests/unit_tests/validators/test_ends_with.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from guardrails.validators import EndsWith, FailResult, PassResult
+
+
+@pytest.mark.parametrize(
+    "input, end, outcome, fix_value",
+    [
+        ("Test string", "g", "pass", None),
```

Comment:
doesn't look like we run null or empty checks in the validator. Can we cover some of those cases in test to make sure they work correctly?